website/docs/Software/OpenBCISoftware/NeuroFly_Toolkit.md

```diff
@@ -46,14 +45,12 @@ This widget contains the tuneable parameters used for the threshold algorithm th
-| Smooth    | Controls how many seconds of data to average over. This reduces the effect of outliers in the dataset. |
-| uV Limit  | Upper limit for the EMG signal.                                                                        |
-| Creep+    | Adjustment speed for upper threshold. How fast it should decrease if not triggered.                    |
-| Creep-    | Adjustment speed for lower threshold. How fast it should increase if not triggered.                    |
-| Min ΔuV   | Minimum difference allowed between upper and lower thresholds.                                         |
-| Low Limit | Minimum lower threshold allowed.                                                                       |
-
-A further description of these parameters can be found on the [GUI Widgets options](../../Software/OpenBCISoftware/02_GUI_Widget_Guide.md/#options) page.
+| Smooth    | This is the size of the window. If we set this value at the smallest setting of 0.01 seconds (ie., lowering the smooth value), our data will be very jittery but responsive. Alternatively, if we increase the smooth and set our window to 2.0 seconds, the output will be very smooth but much less responsive. This reduces the effect of outliers in the dataset. |
+| uV Limit  | This is a cutoff point for an allowable μV value in any individual data block. Any μV values above this number will be chopped off and set to this upper μV limit. This is to prevent erratic blips in the data from substantially distorting the average. Sometimes dropped packets and rapid body movements can create large spikes that don’t correlate to muscle activity. This helps account for those issues. |
+| Creep+    | This value indicates how quickly the upper μV threshold creeps downward. Notice that adjusting this value will effect how fast the the upper threshold decreases if not triggered. We generally recommend this to be slow. If this is too fast and we wait too long between muscle activations, the upper threshold will have crept too close to the lower threshold and the system will be hypersensitive. |
+| Creep-    | This value indicates how quickly the lower μV threshold creeps upward. Notice that adjusting this value will effect how fast the the lower threshold increases if less than the current uV. A lower value enables easier activation, but the signal is more prone to noise. While a large value is the opposite: it is harder to activate, but the signal will be less prone to noise.  |
+| Min ΔuV   | This value sets the minimum voltage range between the upper threshold and the lower threshold. The upper threshold and lower threshold cannot get any closer than this. By making this value bigger, you will have to flex harder to go from 0 to 100% flex, after the thresholds have crept as close as they will get to one another (aka Min ΔμV). |
+| Low Limit | This is the minimum value the lower threshold can be. |
```

Comment:
I would add something like "In general, this value should be set just above the noise floor so that environmental noise does not trigger false activations."